src/svm.py

```diff
@@ -0,0 +1,84 @@
+"""
+file of Support Vector Machines
+"""
+import os
+from datetime import datetime
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
+from sklearn import svm
+from sklearn.metrics import plot_confusion_matrix
+from sklearn.model_selection import GridSearchCV
+
+
+class SupportVectorMachines:  # pylint: disable= R0902
+    """
+    class  Support Vector Machines  classification
+    """
+
+    def __str__(self):
+        return self.__class__.__name__
+
+    def __init__(self, x_data, y_data):
+        """
+            constructor with parameters   of  Support Vector Machines class
+        """
+        self.x_data = x_data
+        self.y_data = y_data
```

Comment:
- You never use these lines: `self.x_data = x_data, self.y_data = y_data`.
- They can be deleted.